task3.py

```diff
@@ -0,0 +1,21 @@
+"""Cover function from 1.3 with tests."""
+from task1_3 import add_to_list_in_dict
+import unittest
+
+
+class FuncTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
it's strange we put all these constants to setup but use in different tests different values. 
usually test is setup, action, assert. your one is just equal

Reply:
solved.